### DISCUSSION TITLE
Capstone - Telebot

### DISCUSSION THREAD

**Discussion by abassimo** [teamalgoritma/community#201] [Q&A] [answered]
Ijin bertanya untuk subplot axes [ apa ada dokumentasi nya  ? ] karena di file sy belum menemukan dengan sample code dengan yang bar / terlewat.

bagaimana mengkoneksikan tabel dfplot kita ke command dibawah , mungkin bisa diberikan 1 contoh.

import matplotlib.pyplot as plt

# prepare 3 subplots vertically
fig, axes = plt.subplots(3, sharex=True, dpi=300)

# create frameless plot
for ax in axes:
    ax.spines['top'].set_visible(False)
    ax.spines['right'].set_visible(False)
    ax.spines['left'].set_visible(False)
    ax.spines['bottom'].set_visible(False)

# first subplot: total spent per age group
axes[0].bar(x=' ', height=' ') color= ' ')
axes[0].set_ylabel( , fontsize= )
![data capture](https://user-images.githubusercontent.com/8667944/178197764-011657a8-72e4-4669-9057-e61e088a7552.JPG)

**Comment by yosiaazarya** [comment, marked as answer]
Terkait dengan hal ini, Bapak hanya perlu untuk mengisi bagian kosong pada rangkaian code yang tersedia. Untuk bagian `axes[0] `, `axes[1] `, dan `axes[2]`  menjelaskan pada barplot pertama, barplot kedua, dan barplot ketiga (sesuai dengan contoh yang ada di notebook) ingin kita buat untuk data apa bapak. Kita hanya perlu melengkapi parameter-parameter yang kosong seperti `x` , `height`, `color`, dan lain-lain sesuai dengan keterangan yang disampaikan. Contoh `x` kita isi dengan index dari dataframe, `height` kita isi dengan data yang kita ingin bandingkan (contoh: cpc). Begitulah yang dapat kami sampaikan, semoga membantu.
